Doc/library/dis.rst

```diff
@@ -861,7 +930,9 @@ iterations of the loop.
 .. opcode:: BUILD_TUPLE (count)
 
    Creates a tuple consuming *count* items from the stack, and pushes the
-   resulting tuple onto the stack.
+   resulting tuple onto the stack.::
+
+      STACK.append(tuple(STACK[-count:]))
```

Comment:
This isn't quite correct. The colon should be before the `-count` and the values need to be popped from the stack.
Also state that `count` should not be zero (it can be, but the compiler emits `LOAD_CONST` in that case, so we might change that it the future.
```
    assert(count > 0)
    STACK, values = STACK[-count:], STACK[:-count]
    STACK.append(tuple(values))
```

Reply:
I am confused. `BUILD_TUPLE` consume the items on top of the stack so if count is 2 that would be `STACK[-2],STACK[-1]  == STACK[-2:]` so shouldn't we write:
```
assert count > 0
STACK, values = STACK[:-count], STACK[-count:]
STACK.append(tuple(values))
```

Reply:
Yes, you're right.